### DISCUSSION TITLE
i use office sk, ask the time now ,it's reply 2020-10,so how can i get the lasted version

### DISCUSSION THREAD

**Discussion by newdots** [waylaidwanderer/node-chatgpt-api#108] [Help]
1. my account is plus 
2. my settings `   modelOptions: {
            // You can override the model name and any other parameters here.
             model: 'text-davinci-003',
        },`

**Comment by newdots** [comment]
if i use reverse proxy, i get this error

`1|server  | Server closed the connection unexpectedly, returning...
1|server  | Error: Server closed the connection unexpectedly. Please make sure you are using a valid access token`

**Comment by newdots** [comment]
but i have setting from the https://chat.openai.com/api/auth/session get accessToken   to  openaiApiKey. just follow the readme ,why thanks

**Comment by waylaidwanderer** [comment]
`modelOptions` is for using `ChatGPTClient`, not `ChatGPTBrowserClient`. Set your access token in `chatGptBrowserClient.accessToken` in `settings.js`. You don't need to set `openaiApiKey` at all.

Example (`settings.js` and then `npm run server`):
```JS
module.exports = {
    chatGptBrowserClient: {
        reverseProxyUrl: 'https://chatgpt.duti.tech/api/conversation',
        accessToken: 'access token',
    },
    // Options for the API server
    apiOptions: {
        port: process.env.API_PORT || 3000,
        host: process.env.API_HOST || 'localhost',
        clientToUse: 'chatgpt-browser',
    },
}
```
Example (using `ChatGPTBrowserClient` directly):
```JS
new ChatGPTBrowserClient({
  reverseProxyUrl: 'https://chatgpt.duti.tech/api/conversation',
  accessToken: 'access token',
});
```

**Comment by newdots** [comment]
i have this settings , comment the openApiKey.
<img width="1421" alt="image" src="https://user-images.githubusercontent.com/114389694/220504688-a4aaec17-b137-44e3-99e2-71c3a0bbcd55.png">
still can't access ,this is the debug log
<img width="970" alt="image" src="https://user-images.githubusercontent.com/114389694/220504948-fcffdc97-bcaa-438f-8f52-7d38208dd4f6.png">

**Reply by waylaidwanderer** [reply]
Why is the conversation ID "kefu2"? If you're starting a new conversation, it must be "null" instead. It's probably giving you that error because the conversation ID does not exist in chat.openai.com.

**Reply by newdots** [reply]
ok,you are right.thank you .but i notice the speed user reverse is lower little
